Utils.py

```diff
@@ -933,7 +933,7 @@ def freeze_support() -> None:
 
 def visualize_regions(root_region: Region, file_name: str, *,
                       show_entrance_names: bool = False, show_locations: bool = True, show_other_regions: bool = True,
-                      linetype_ortho: bool = True) -> None:
+                      linetype_ortho: bool = True, regions_to_highlight: set[Region] = None) -> None:
```

Comment:
```suggestion
                      linetype_ortho: bool = True, regions_to_highlight: set[Region] | None = None) -> None:
```
pep 484 says:
> A past version of this PEP allowed type checkers to assume an optional type when the default value is None, as in this code:
> 
> `def handle_employee(e: Employee = None): ...`
> 
> This would have been treated as equivalent to:
> 
> `def handle_employee(e: Optional[Employee] = None) -> None: ...`
> 
> This is no longer the recommended behavior. Type checkers should move towards requiring the optional type to be made explicit.